Agile Development/Burndown Chart/requirements.txt

```diff
@@ -0,0 +1,22 @@
+certifi==2024.8.30
+charset-normalizer==3.4.0
+contourpy==1.3.0
+cycler==0.12.1
+fonttools==4.54.1
+idna==3.10
+importlib_resources==6.4.5
+jpholiday==0.1.10
+kiwisolver==1.4.7
+matplotlib==3.9.2
+numpy==2.0.2
+packaging==24.1
+pandas==2.2.3
+pillow==11.0.0
+pyparsing==3.2.0
+python-dateutil==2.9.0.post0
+pytz==2024.2
+requests==2.32.3
+six==1.16.0
+tzdata==2024.2
+urllib3==1.26.13
+zipp==3.20.2
```

Comment:
there's some extra things that got pulled in here that you shouldn't need and aren't using, like kiwi solver. Can you just manually remove these?

Reply:
Removed unnecessary libraries.